packages/typespec-test/test/openai_modular/generated/typespec-ts/src/models/models.ts

```diff
@@ -610,3 +610,23 @@ export type ImageSize = string;
 /** The format in which the generated images are returned. */
 /** "url", "b64_json" */
 export type ImageGenerationResponseFormat = string;
+
+/** Optional settings to control how fields are processed when using a configured Azure Cognitive Search resource. */
+export interface AzureCognitiveSearchIndexFieldMappingOptions {
```

Comment:
Could you help me understand why the model is newly added? did we support @usage or @access already?